### DISCUSSION TITLE
Human whole blood cell scRNA-seq data

### DISCUSSION THREAD

**Discussion by beginner984** [satijalab/seurat#5341] [Ideas]
Hello

Seurat team has a data base of human PBMC here

https://app.azimuth.hubmapconsortium.org/app/human-pbmc

We can annotate single cell PBMC against that to immune cell types

But I noticed we can not profile polymorphonuclear leukocytes (PMN) like Neutrophils by PBMC

Do you know any data base for human single cell data (preferably cancers) but from `whole blood cells` not PBMC?

Thank you so much in advance

**Comment by samuel-marsh** [comment]
I would suggest checking out Immgen databases.  They have both single cell and bulk expression data profiles from wide variety of methods and populations.

https://www.immgen.org/Databrowser19/DatabrowserPage.html

Best,
Sam
